pydrive2/test/README.rst

```diff
@@ -1,13 +1,30 @@
-Instructions
-------------
-
--  Install timeout-decorator and futures (used for parallel upload and
-download testing).
--  Create *configs* and *credentials* folder in working directory.
--  Put *client_secrets.json* file from API console into *configs* folder.
--  Put *clients_secrets_local.json* from API console, for native device in
-*configs* folder.
--  Put different *a.png* and *b.png* file in working directory.
--  Replace {{ }} sections in *settings/test2.yaml* with the relevant sections
-from your config file.
--  For ServiceAuth test, place PKCS12 file in working directory.
+Run tests locally
```

Comment:
could you please run a prettier or VSCode wrapper on it? It's better to have it wrapped at 80 symbols

Reply:
Have used `rewrap` VSCode plugin.